lib/mongoose_push_web/schemas/request/send_notification/flat.ex

```diff
@@ -0,0 +1,32 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Flat do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "APIv1.Request.Push",
+    description: "Push notification request schema",
+    type: :object,
+    properties: %{
```

Comment:
As far as I know `service`, `body` and `title` are required parameters. I think that `OpenApiSpex.Schema` allows to mark certain fields as required by providing a list of required fields like below:
```suggestion
    required: [:service, :body, :title],
    properties: %{
```

Reply:
Oh, I didn't notice. It's added below `properties`. That's ok :)